### PR TITLE
refactor: [DD-3098] Refactor theme colors

### PR DESCRIPTION
**Pull request by MorganaLF**
Related PR: https://github.com/akropolisio/akropolis-frontend/pull/982

### CODE REVIEW

File: packages/styles/src/theme.ts [hunk 1]
```diff
@@ -227,6 +243,22 @@ export const darkPalette = {
...
     paperSecondary: colors.darkBlueMagenta,
     paperLight: colors.jaguar,
+    paperExtraLight: colors.cinder,
+    amountField: colors.charadeLight,
+    containedButton: colors.blackRussian,
+    warning: colors.blackRussian,
+    card: colors.steelGray,
+    productCard: colors.darkSpace,
+    tokenBadge: colors.mulledWine,
+    body: colors.obsidian,
```
Comment (imin314) [inline packages/styles/src/theme.ts:253]: body colors match the default colors

File: packages/styles/src/theme.ts [hunk 2]
```diff
@@ -227,6 +243,22 @@ export const darkPalette = {
     paper: colors.cinder,
     paperSecondary: colors.darkBlueMagenta,
     paperLight: colors.jaguar,
+    paperExtraLight: colors.cinder,
+    amountField: colors.charadeLight,
+    containedButton: colors.blackRussian,
+    warning: colors.blackRussian,
+    card: colors.steelGray,
+    productCard: colors.darkSpace,
```
Comment (imin314) [inline packages/styles/src/theme.ts:251]: I think we should avoid such specific names in the akroweb theme. Same for the "poolAddress". Can we make it more general?

File: packages/styles/src/theme.ts [hunk 3]
```diff
@@ -198,6 +198,22 @@ const lightPalette = {
     paper: colors.athensGray,
     paperSecondary: colors.athensGray,
     paperLight: colors.white,
+    paperExtraLight: colors.white,
+    amountField: colors.athensGray,
```
Comment (imin314) [inline packages/styles/src/theme.ts:202]: The amountField background is used locally inside the AmountField component. I think it doesn't look like a case for extending the palette.

File: packages/styles/src/theme.ts [hunk 4]
```diff
@@ -227,6 +243,22 @@ export const darkPalette = {
     paper: colors.cinder,
     paperSecondary: colors.darkBlueMagenta,
     paperLight: colors.jaguar,
+    paperExtraLight: colors.cinder,
+    amountField: colors.charadeLight,
+    containedButton: colors.blackRussian,
+    warning: colors.blackRussian,
+    card: colors.steelGray,
+    productCard: colors.darkSpace,
+    tokenBadge: colors.mulledWine,
```
Comment (imin314) [inline packages/styles/src/theme.ts:252]: Is it possible to shorten it to more general "badge"?

**Review by imin314 — APPROVED** [review]

**Review by in19farkt — APPROVED** [review]

**Pull request merged**